cpp/test/cpu_operations_test/Inverse_test.cc

```diff
@@ -47,7 +47,7 @@ class InverseTest : public ::testing::Test {
 };
 
 
-typedef ::testing::Types<int, float, double> MyTypes;
+typedef ::testing::Types</*int,*/float, double> MyTypes;
```

Comment:
Interesting I had that taken out I must have commited my latest version but not pushed it to my repository to reflect the changes. Sorry!